website/source/guides/operations/vault-integration/index.html.md

```diff
@@ -557,18 +575,24 @@ There are a few key points to note here:
 
 - We have specified the `access-tables` policy in the [vault][vault-jobspec]
   stanza of this job. The Nomad client will receive a token with this policy
-attached. Recall from the previous step that this policy will allow our
-application to read from the `database/creds/accessdb` endpoint in Vault and retrieve credentials.
+  attached. Recall from the previous step that this policy will allow our
+  application to read from the `database/creds/accessdb` endpoint in Vault and
+  retrieve credentials.
 - We are using the [template][template] stanza's [vault
   integration][nomad-template-vault] to populate the JSON configuration file
-that our application needs. The underlying tool being used is [Consul
-Template][consul-template]. You can use Consul Template's documentation to learn
-more about the [syntax][consul-temp-syntax] needed to interact with Vault.
-Please note that although we have defined the template [inline][inline], we can use the template stanza [in conjunction with the artifact stanza][remote-template] to download an input template from a remote source such as an S3 bucket.
+  that our application needs. The underlying tool being used is [Consul
+  Template][consul-template]. You can use Consul Template's documentation to
+  learn more about the [syntax][consul-temp-syntax] needed to interact with
+  Vault. Please note that although we have defined the template
+  [inline][inline], we can use the template stanza [in conjunction with the
+  artifact stanza][remote-template] to download an input template from a remote
+  source such as an S3 bucket.
+- We are using the `toJSON` function to ensure Nomad passes the password to task
+  unchanged. You can see [go-envparser's README][go-envparser] for more details.
```

Comment:
This sentence would be true if you're creating an environment variable template (`env=true`), but since you're creating a JSON config file `toJSON` ensures the variable is encoded as a JSON string. Perhaps a better explanation would be:

> We are using the `toJSON` function to ensure the password is encoded as a JSON string. Any templated value which may contain special characters (like quotes or newlines), should be passed through the `toJSON` function.

go-envparser is only used when `env=true`